en-us/config.md

```diff
@@ -1,8 +1,8 @@
-# 配置说明
+# Configuration
 
-默认的创世块和配置样例在 `devtools/chain/` 文件夹中，此处对其中的一些字段进行说明。
+Default genesis block and config example is in `devtools/chain/` Here's some explanation to it.
```

Comment:
“Default genesis block configuration and chain configuration....”

Kind of inaccurate in Chinese version，I‘ll fix it